### ISSUE TITLE
Add a "New Article Published" email notification to support continuous publication

### ISSUE THREAD

**Issue by jmacgreg** [pkp/pkp-lib#4504]
In OJS 3, editors can choose to send a "New Issue" notification to readers on the publication of an issue, but there isn't a similar option when an article has been published against an already-published issue, ie. for continuous publication workflows. I could see this being developed in one of two ways: 

1) add a "Send notification email to all registered users" option to the article Schedule for Publication page, active and visible only if the selected issue is already published. This should use a unique "New Article Published" email template. It should also have an accompanying user profile notification management option. 

2) re-add the more general Notify Users functionality. This has been discussed before - see #4017, and also multiple comments in #2561.

**Comment by jmacgreg** [comment]
Note: #2 above can probably be satisfied by using the Announcements function in OJS, and probably shouldn't be considered. I think there's value in considering #1 though.

**Comment by openacademia** [comment]
Yes! This would be a very welcomed feature! The first option sounds great. 

Using the Announcements function to communicate with readers about new articles being published is not very good as it sends a very generic notification without any clues about the new articles. The reader has to first click on a link in the email to come to the announcement and then click on another link from the actual announcement to reach the new article. And it is not possible to customize the notification template with a direct link to the Current Issue page, since it is the same notification used for new submissions etc.

The first option suggested here would be fantastic! It would be good if somewhere in the settings, one could select Issue Publication vs. Continuous Publication - and for that to govern how the notifications are used (new issue vs new article) but also whether the journal home says "Current Issue" (grrrr!) or "Current Volume". But maybe that's a bigger overhaul that is not possible at the moment. Finding a sensible way to communicate to reader that a new article has been published (without forcing the reader to click on multiple links to get to said article) would be a fantastic improvement either way!

**Comment by ajnyga** [comment]
Just a note that I am probably diving into continuous publishing and will introduce a possibility to publish without issues altogether. Still waiting for some answers to couple of questions from Alec and the big merge of versioning before working on it. Just finished talking with Bozana what it means for DOIs. This would of course be considered as well.

For this issue regarding point 2 and using announcements: the big problem is that the email of a new announcement does not contain the announcement text itself. That is why many of our journals do not like to use it at all.

**Comment by jmacgreg** [comment]
See also #1407. @ajnyga, should I close this ticket in favour of that one?

**Comment by ajnyga** [comment]
Once the discussion around versioning etc. come the a certain point, I will create create two issues. One for preprint serve work and one for continuous publishing work and try to link all these issues to those two. Maybe we do not need to close the for now, but will try to create a list to one place.

**Comment by unkej** [comment]
Is there anything in progress for notifications for continuous publishing? Couldn't find a more up to date issue.

**Comment by kshuttle** [comment]
Here is some suggested email template text for announcing a new item publishing under a continuous publishing model:

Suggested title for new template: ITEM_PUBLISH_CONTINUOUS

Sent by: Editor
Received by: Registered readers
Should the sender be able to preview and customize this email? Yes
Should the recipient be able to opt out of receiving this email? Yes

The following has been drafted for this new email template:

**Email subject**

Publication announcement: A new item has been published

**Email body**

Dear Reader,

We are pleased to announce that a new item has been published in {$contextName}. We invite you to view this [item](link) and see other articles in [this issue](link).

[{$submissionTitle}](link)
{$submissionAuthors}

{$submissionAbstract}

Sincerely,

{$signature}

**Email description**

This email is sent to registered readers via the \”Notify Users\” link in the Editor’s User Home. It notifies readers of a new item that is published separately from an issue, using a continuous publishing model. 

**The following work needs to be done:**

 - [ ] Add this to OJS
